app/models/mixins/miq_request_mixin.rb

```diff
@@ -15,6 +15,10 @@ def display_message(default_msg = nil)
 
   def user_message=(msg)
     msg = msg.truncate(255)
+    self.user_message_untruncated = msg
```

Comment:
@paulslaby isn't line number 17 truncating the message and updating the msg variable which is getting used in line 18? so line number 18 is setting a truncated message

Reply:
I believe this is just a refactoring to allow sub-classes to override the `user_message=` logic (in this case skip the `truncate` call) but still call the same `user_message_untruncated=` method to save the data.

Since the 255 limit was removed I still not convinced any of this is needed in favor of removing the `truncate` call.